src/monitord/main.c

```diff
@@ -156,9 +156,15 @@ int main(int argc, char **argv)
     mond.emailfrom = NULL;
     mond.emailidsname = NULL;
 
+    /* Setting default agent's global configuration */
+    mond.global.agents_disconnection_time = 20;
```

Comment:
We can move this initialization, together with the previous ones, and ReadConfig to a monitord config method, as in other modules. Like RemotedConfig()

Reply:
+1

Reply:
Now there is a **MonitordConfig()** that reads the configuration